WeakAuras/BuffTrigger2.lua

```diff
@@ -2502,14 +2502,44 @@ local function EqualZero(x)
   return x == 0
 end
 
+local function InitProblems()
+  return {
+    untrackableSoftTarget = {
+      severity = "info",
+      message = L["A trigger in this aura is set up to track a soft target unit, but you don't have the CVars set up for this to work correctly. Consider either changing the unit tracked, or configuring the Soft Target CVars."],
+      flagged = false,
+      check = function(trigger)
+        return WeakAuras.IsUntrackableSoftTarget(trigger.unit)
+      end
+    }
+  }
+end
+
+local function CheckProblems(trigger, problems)
+  for _, problem in pairs(problems) do
+    if not problem.flagged and problem.check(trigger) then
+      problem.flagged = true
+    end
+  end
+end
+
+local function PublishProblems(problems, uid)
+  for key, problem in pairs(problems) do
+    if problem.flagged then
+      Private.AuraWarnings.UpdateWarning(uid, key, problem.severity, problem.message, problem.printOnConsole)
+    end
```

Comment:
else
  Private.AuraWarnings.UpdateWarning(uid, key)
end

That should clear the problems if you edit the aura.